output/drivers/pygame_emulator_factory.py

```diff
@@ -6,12 +6,11 @@
 """
 
 import luma.emulator.device
+
 from helpers import setup_logger
 
 # ignore PIL debug messages
 
-logging.getLogger("PIL").setLevel(logging.ERROR)
```

Comment:
This fixes the import error, but loses a feature - namely, PIL library logging output suppression (which is pointed out in the comment right above). Sorry for not pointing this out earlier =)

Reply:
Also, we don't even use the resulting logger - so, removed